etc/adminhtml/system.xml

```diff
@@ -1518,15 +1522,23 @@
                 <field id="write_timeout" translate="label comment" type="text" sortOrder="110" showInDefault="1">
                     <label>Write Timeout (In Seconds)</label>
                 </field>
+                <field id="forward_to_replicas" translate="label comment" type="select" sortOrder="115" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
This advanced tab is becoming very messy, maybe we could add a task to reorganize it a little bit, WDYT ?

Reply:
I'm all for that. Will spin up in Jira.